gallery/gallery-library/src/main/java/org/silverpeas/components/gallery/MediaUtil.java

```diff
@@ -109,14 +107,14 @@ public static void processSound(final FileHandler fileHandler, Sound sound,
    * @param uploadedFile the current uploaded sound
    * @throws Exception
    */
-  public static void processSound(final FileHandler fileHandler, Sound sound,
+  public synchronized static void processSound(final FileHandler fileHandler, Sound sound,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)